src/views/Scenario/Scenario.js

```diff
@@ -140,6 +142,13 @@ const Scenario = (props) => {
               scenarioName={currentScenario.data?.name}
               scenarioState={currentScenario.data?.state}
               noScenario={noScenario}
+              scenarioId={currentScenario.data?.id}
+              downloadLogsFile={() => {
+                ScenariorunService.downloadLogsFile(
+                  currentScenario.data.lastRun,
```

Comment:
Not sure if this is necessary, but `currentScenario.data?.lastRun` (with a question mark) would be more consistent with the lines above

Reply:
Right. Fixed

Reply:
Fixed